tests/network/vmi_networking.go

```diff
@@ -270,6 +288,9 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 				&expect.BSnd{S: fmt.Sprintf(findCmd, masqIface.PciAddress) + console.CRLF},
 				// https://admin.pci-ids.ucw.cz/read/PC/8086
 				&expect.BExp{R: "0x8086"},
+				&expect.BSnd{S: fmt.Sprintf(findCmd, bridgeIface.PciAddress) + console.CRLF},
+				// https://admin.pci-ids.ucw.cz/read/PC/1af4
+				&expect.BExp{R: "0x1af4"},
```

Comment:
Please consider naming `0x8086` and `0x1af4` by using consts.
It would also allow moving the comments above the consts.